pkg/deploy/generator/rp.go

```diff
@@ -346,6 +357,12 @@ chmod 0600 /etc/mdsd.pem
 
 az logout
 
+mkdir /etc/rp
+ADMIN_API_CA_BUNDLE_PATH=/etc/rp/admin-ca-bundle.pem
```

Comment:
I am ok with hard-coding this in the codebase rather than adding another env var.  Please can we standardise on /etc/aro-rp ?